experiments/experiments-impl/src/main/java/com/duckduckgo/experiments/impl/ExperimentFiltersManager.kt

```diff
@@ -51,6 +58,11 @@ class ExperimentFiltersManagerImpl @Inject constructor(
             val userAndroidVersion = appBuildConfig.sdkInt.toString()
             filters[ANDROID_VERSION] = entity.filters!!.androidVersion.contains(userAndroidVersion)
         }
+        if (entity.filters?.privacyProEligible != null) {
+            runBlocking(dispatcherProvider.io()) {
```

Comment:
I'd probably just wrap the suspend call rather than the whole thing and do something like:

```
val isEligible =  runBlocking(dispatcherProvider.io()) { subscriptions.isEligible() }
filters[PRIVACY_PRO_ELIGIBLE] = entity.filters?.privacyProEligible == isEligible
```